src/taco-utils/telemetry.ts

```diff
@@ -205,6 +206,10 @@ module TacoUtility {
             private static INTERNAL_USER_ENV_VAR: string = "TACOINTERNAL";
 
             private static get telemetrySettingsFile(): string {
+                if (Telemetry.appName === "REMOTE_BUILD") {
```

Comment:
This feels a little too tightly coupled to me. Perhaps we should have telemetry.init accept an optional settings filename, and leave TelemetrySettings.json as a default?